### DISCUSSION TITLE
Roadmap page for Mujoco

### DISCUSSION THREAD

**Discussion by MyelinsheathXD** [google-deepmind/mujoco#482] [Ideas]
I am interested long roadmaps of Mujoco engine.
Big companies products like Unity, Unreal engines have their own roadmap pages like these:

https://unity.com/roadmap/unity-platform/dots

https://portal.productboard.com/epicgames/1-unreal-engine-public-roadmap/tabs/80-unreal-engine-5-1-in-progress

Will there be any roadmap page in future ?

**Comment by yuvaltassa** [comment]
That is a good idea.

We'll discuss internally what is the best way to do that.

Thanks!

**Reply by driesmarzougui** [reply]
This would still be very nice to have. 😉

**Comment by imkarthikk** [comment]
This is a great idea. Would love to suggest Hellonext (https://hellonext.co) for roadmapping. It's free for open source software as well.
